python/lsst/ts/observatory/control/auxtel/atcalsys.py

```diff
@@ -196,11 +196,38 @@ async def prepare_for_flat(self, config_name: str) -> None:
             else utils.make_done_future()
         )
 
+        task_setup_electrometer = self.setup_electrometers(
+            mode=int(config_data["electrometer_mode"]),
+            range=float(config_data["electrometer_range"]),
+            integration_time=float(config_data["electrometer_integration_time"]),
+        )
+
         await asyncio.gather(
-            task_setup_monochromator,
-            task_setup_latiss,
+            task_setup_monochromator, task_setup_latiss, task_setup_electrometer
         )
 
+    async def setup_electrometers(
+        self, mode: int = 1, range: float = -1, integration_time: float = 0.1
```

Comment:
do you really need the default values here? it seems like this is always getting values from `config_data` above. I suggest you remove the default values.